packages/@okta/vuepress-site/docs/concepts/policies/index.md

```diff
@@ -12,54 +12,86 @@ Policies are evaluated when a request is made. Rules in the policies define perm
+* [Profile enrollment policy](https://help.okta.com/okta_help.htm?type=oie&id=ext-pe-policies): Collects the attributes required to validate end users when they attempt to access your app. You can use this policy for [self-service registration](/docs/guides/oie-embedded-sdk-use-case-self-reg/android/main/) or for [progressive enrollment](https://help.okta.com/okta_help.htm?type=oie&id=ext-pe-policies). With self-service registration flows, end users can register and activate their profiles by clicking a sign-up link in the Sign-In Widget or through a custom embedded authentication solution. With progressive enrollment flows, you can capture the minimum user information required to create a profile and then continually build out those user profiles during subsequent sign-in operations. You can control what information is collected, validate those input values, and [trigger inline hooks](/docs/guides/registration-inline-hook/nodejs/main/).
 
-* [Password Policy](/docs/reference/api/policy/#password-policy):  Determines the requirements for a user's password length and complexity, as well as the frequency with which a password must be changed. This policy also governs the recovery operations that may be performed by the user, including change password, reset (forgot) password, and self-service password unlock.
+### API access policy
 
-* [IdP Discovery Policy](/docs/reference/api/policy/#idp-discovery-policy): Determines where to route users when they are attempting to sign in to your org. Users can be routed to a variety of [identity providers](/docs/guides/add-an-external-idp/).
+* [OAuth Authorization Policy](/docs/reference/api/authorization-servers/#policy-object): Manages authorization between clients and Okta. The access policy is specific to a particular client application and the rules that it contains define particular token lifetimes for a given combination of grant type, user, and scope.
 
-> **Note:** This policy isn't for performing authentication or authorization. It is used only to determine where a user should be routed. You can't control access with an IdP Discovery Policy.
+### Routing rule
 
-* [OAuth Authorization Policy](/docs/reference/api/authorization-servers/#policy-object): Specific to a particular client application and the rules that it contains define particular token lifetimes for a given combination of grant type, user, and scope.
+* [IdP Discovery Policy](/docs/reference/api/policy/#idp-discovery-policy): Determines where to route users when they attempt to sign in to your org. You can route users to a variety of [identity providers](/docs/guides/add-an-external-idp/).
+
+  > **Note:** This policy isn't for performing authentication or authorization. It’s used only to determine where a user is routed. You can't control access with an IdP Discovery Policy.
+
+## About authentication policies
```

Comment:
I felt a bit befuddled as I came to this section. I asked myself, "Didn't i already read about "Authentication policies" under "Sign-on policies"? Why does this section get called out here?

Reply:
because it describes in more detail what authentication policies do. I'll try again.